src/App.js

```diff
@@ -1,16 +1,62 @@
-import React from 'react';
+import React, {Component} from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+
+let storage = window.localStorage;
+let storageList = [];
+if (storage.list !== undefined){
+	storageList = JSON.parse(storage.list);
+}
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
-  render() {
-    return (
-      <div>
-        <h2>Welcome to your Todo App!</h2>
-      </div>
-    );
-  }
+constructor() {
+  super();
+  this.state = {todo: storageList, current: ''}
+}
+
+handleInputChange = event => {
+  this.setState({current: event.target.value});
+}
+
+addTodo = (event, prevState) => {
+  event.preventDefault();
+  let clear = document.querySelector('.toDoForm input');
```

Comment:
Good job getting around whatever problem you were facing, but don't use querySelectors in React